src/CommunityToolkit/Xamarin.CommunityToolkit/Helpers/SafeFireAndForgetExtensions.shared.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Threading.Tasks;
+
+// Inspired by https://github.com/brminnick/AsyncAwaitBestPractices
+namespace Xamarin.CommunityToolkit.Helpers
+{
+	/// <summary>
+	/// Extension methods for System.Threading.Tasks.Task and System.Threading.Tasks.ValueTask
+	/// </summary>
+	static class SafeFireAndForgetExtensions
+	{
+		/// <summary>
+		/// Safely execute the ValueTask without waiting for it to complete before moving to the next line of code; commonly known as "Fire And Forget". Inspired by John Thiriet's blog post, "Removing Async Void": https://johnthiriet.com/removing-async-void/.
+		/// </summary>
+		/// <param name="task">ValueTask.</param>
+		/// <param name="onException">If an exception is thrown in the ValueTask, <c>onException</c> will execute. If onException is null, the exception will be re-thrown</param>
+		/// <param name="continueOnCapturedContext">If set to <c>true</c>, continue on captured context; this will ensure that the Synchronization Context returns to the calling thread. If set to <c>false</c>, continue on a different context; this will allow the Synchronization Context to continue on a different thread</param>
+		public static void SafeFireAndForget(this ValueTask task, in Action<Exception>? onException = null, in bool continueOnCapturedContext = false) => HandleSafeFireAndForget(task, continueOnCapturedContext, onException);
```

Comment:
I guess we can convert all these methods to "internal". That's not a big deal since the class is internal, but just a note :)

Reply:
Also, I appreciate if you can move the method's body to the new line for better readability 👍

Reply:
Great idea! No problem!